common/src/main/java/com/wynntils/screens/settings/elements/CustomColorConfigOptionElement.java

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright © Wynntils 2022.
+ * This file is released under AGPLv3. See LICENSE for full license details.
+ */
+package com.wynntils.screens.settings.elements;
+
+import com.mojang.blaze3d.vertex.PoseStack;
+import com.wynntils.core.config.ConfigHolder;
+import com.wynntils.mc.objects.CommonColors;
+import com.wynntils.mc.objects.CustomColor;
+import com.wynntils.mc.render.RenderUtils;
+import com.wynntils.screens.settings.WynntilsSettingsScreen;
+import com.wynntils.screens.settings.widgets.FeatureSettingWidget;
+
+public class CustomColorConfigOptionElement extends TextConfigOptionElement {
+    public CustomColorConfigOptionElement(
+            ConfigHolder configHolder,
+            FeatureSettingWidget featureSettingWidget,
+            WynntilsSettingsScreen settingsScreen) {
+        super(configHolder, featureSettingWidget, settingsScreen);
+    }
+
+    @Override
+    protected void renderSuccessState(PoseStack poseStack) {
+        poseStack.pushPose();
+
+        poseStack.translate(-getTextInputHeight() * 1.2f, 0, 0);
+
+        if (!this.lastParseSuccessful) {
+            RenderUtils.drawLine(
```

Comment:
Maybe add comment "make a red cross if parsing failed" or something like that. Took me a while to figure out.